### PR TITLE
Simplifies the test setup a tiny bit

### PR DESCRIPTION
**Pull request by Willianvdv**
Almost not PR worthy, but while debugging the issues on #773 I stumbled upon this minor improvement.

### CODE REVIEW

File: spec/generators/graphql/install_generator_spec.rb [hunk 1]
```diff
@@ -3,20 +3,19 @@
...
 
 class GraphQLGeneratorsInstallGeneratorTest < Rails::Generators::TestCase
-
   tests Graphql::Generators::InstallGenerator
   destination File.expand_path("../../../tmp/dummy", File.dirname(__FILE__))
 
   setup do
     prepare_destination
-    FileUtils.cd(File.expand_path("../../../tmp", File.dirname(__FILE__))) do
-      `rm -rf dummy`
```
Comment (Willianvdv) [inline spec/generators/graphql/install_generator_spec.rb]: prepare_destination already removes the dummy app

**Comment by rmosolgo** [comment]
👌  nice! just curious, did you find a good resource to learn generator tests, or have you been learning as you go? I've struggled to find a proper way to learn :S

**Comment by Willianvdv** [comment]
My resource is this Gem 😆

File: spec/generators/graphql/install_generator_spec.rb [hunk 2]
```diff
@@ -3,20 +3,19 @@
...
+
+    FileUtils.cd(File.join(destination_root, '..')) do
       `rails new dummy --skip-active-record --skip-test-unit --skip-spring --skip-bundle`
     end
   end
```
Comment (ThisIsMissEm) [inline spec/generators/graphql/install_generator_spec.rb]: I think it'd be good to get this into base generator test or something. It's common to all our tests

**Comment by ThisIsMissEm** [comment]
@rmosolgo @williamvdv I've been learning from the Rails, Devise and Omniauth generators.

File: spec/generators/graphql/install_generator_spec.rb (hunk 2 above)
Comment (Willianvdv) [inline spec/generators/graphql/install_generator_spec.rb]: yes!

Comment (rmosolgo) [inline spec/generators/graphql/install_generator_spec.rb]: It's really slow to do this! 

If we have a lot of tests that need a Rails app, we should keep a dummy app in `spec/dummy`, this has worked well for react-rails anyways

Comment (Willianvdv) [inline spec/generators/graphql/install_generator_spec.rb]: I thought you meant some spec helper that you can call when dealing with a spec that needs Rails power.

Comment (rmosolgo) [inline spec/generators/graphql/install_generator_spec.rb]: :+1:

**Pull request merged**